mobile_rewards/src/server.rs

```diff
@@ -1,34 +1,42 @@
 use crate::{
     datetime_from_epoch,
     error::DecodeError,
-    follower::FollowerService,
     keypair::Keypair,
     meta::Meta,
     pending_txn::{PendingTxn, Status},
-    subnetwork_rewards::{construct_txn, get_time_range, RewardPeriod, SubnetworkRewards},
     traits::B64,
     transaction::TransactionService,
     txn_status::TxnStatus,
     Error, Result,
 };
-use chrono::{Duration, Utc};
-use futures::try_join;
+use chrono::{DateTime, Duration, Utc};
+use futures::{stream, try_join, StreamExt};
 use helium_proto::{
     blockchain_txn::Txn,
     services::{
-        follower::FollowerTxnStreamRespV1,
+        follower::{
+            self, FollowerSubnetworkLastRewardHeightReqV1, FollowerTxnStreamReqV1,
+            FollowerTxnStreamRespV1,
+        },
+        poc_mobile::OwnerEmissions,
         transaction::{TxnQueryRespV1, TxnStatus as ProtoTxnStatus},
+        Channel, Endpoint,
     },
-    BlockchainTokenTypeV1, BlockchainTxn, BlockchainTxnSubnetworkRewardsV1, Message,
+    BlockchainTokenTypeV1, BlockchainTokenTypeV1 as ProtoTokenType, BlockchainTxn,
```

Comment:
```suggestion
   BlockchainTokenTypeV1 as ProtoTokenType, BlockchainTxn,
```
looks like a redundant import of token type